DuckDuckGo/Tab/TabExtensions/SpecialErrorPageTabExtension.swift

```diff
@@ -199,17 +204,16 @@ extension SpecialErrorPageTabExtension: SpecialErrorPageUserScriptDelegate {
 
     // Special error page "Leave site" button action
     func leaveSiteAction() {
-        guard webView?.canGoBack == true else {
-            webView?.close()
-            return
+        Task { @MainActor in
+            await self.webView?.openNewTabFromErrorPage()
```

Comment:
I don’t think SSL navigation was causing problems…
do we want to generalise this behaviour for all?